### PR TITLE
Cql4bonnie qdm5 3 937

### PR DESCRIPTION
**Pull request by losborne**
JIRA Test: https://jira.mitre.org/browse/BONNIE-961

### CODE REVIEW

File: app/assets/javascripts/cqlpatient.js.coffee [hunk 1]
```diff
@@ -183,8 +183,13 @@ class CQL_QDM.CQLPatient
...
           # EncounterPerformed
-          classname = dc.description.substr(0, dc.description.indexOf(':'))
-          classname = classname.replace(/,/g, '').replace(/\//g, '')
+          # TODO: this needs to be modified to look at the HQMF template OID instead of the description
+          classname = dc.description.substr(0, dc.description.lastIndexOf(':'))
+          # remove commas, slashes, and colons
+          classname = classname.replace(/,/g, '').replace(/\//g, '').replace(/:/g, '')
+          # make all words start upper case
+          classname = classname.replace(/\w\S*/g, (txt) ->
+            txt.charAt(0).toUpperCase() + txt.substr(1))
```
Comment (jbradl11) [inline app/assets/javascripts/cqlpatient.js.coffee]: Why did this change? I am just curious.

Comment (jbradl11) [inline app/assets/javascripts/cqlpatient.js.coffee]: Can you fix that? Just to make things more clear.

File: app/assets/javascripts/datatypes/encounter.js.coffee [hunk 2]
```diff
@@ -118,11 +118,11 @@ class CQL_QDM.EncounterPerformed extends CQL_QDM.QDMDatatype
     super @entry
     @_admissionSource = @entry.admission_source
     @_authorDatetime = CQL_QDM.Helpers.convertDateTime(@entry.start_time)
-    @_diagnosis = @entry.diagnosis
+    @_diagnoses = @entry.diagnosis
     @_dischargeDisposition = @entry.dischargeDisposition
     @_locationPeriodLow = CQL_QDM.Helpers.convertDateTime(@entry.facility?['start_time'])
     @_locationPeriodHigh = CQL_QDM.Helpers.convertDateTime(@entry.facility?['end_time'])
-    @_facilityLocation = @entry.facility?.code
+    @_facilityLocations = @entry.facility?.code
```
Comment (edeyoung) [inline app/assets/javascripts/datatypes/encounter.js.coffee]: I'd hold on this change until the facility stuff gets worked out. we also need to double check the new model info file to see how they're managing the naming and referencing for facility given that it looks like it's a facility object with child location and start/end times.

File: app/assets/javascripts/cqlpatient.js.coffee (hunk 1 above)
Comment (edeyoung) [inline app/assets/javascripts/cqlpatient.js.coffee]: i've merged in the cql4bonnie changes. the way github does diffs for this is based off of the common ancestor so it's continuing to show this in the diff even though it's not actually different (fun, huh?)

**Pull request merged**